cypress/support/step-definitions/detail-steps.js

```diff
@@ -1,5 +1,10 @@
-import { childrenPreview, footnotePreview, iconSelect } from '../../locators/detail';
-import { icon } from '../../locators';
+import {
+  childrenPreview,
+  footnotePreview
+} from '../../locators/detail';
+import {
+  iconNoIframe
```

Comment:
please, change the name as we discussed `icon` for `noIframe` and `iconIframe` for `iFrame`